### PR TITLE
sync with english version for `tabs.query()`

### PR DESCRIPTION
**Pull request by yin1999**
### Description

sync with english version for `tabs.query()`

### Related issues and pull requests

https://github.com/orgs/mdn/discussions/402

### CODE REVIEW

File: files/zh-cn/mozilla/add-ons/webextensions/api/tabs/query/index.md [hunk 1]
```diff
@@ -1,151 +1,146 @@
...
-    // tab.url requires the `tabs` permission
+  for (const tab of tabs) {
+    // tab.url 需要 `tabs` 权限或匹配的主机权限。
     console.log(tab.url);
   }
 }
 
 function onError(error) {
-  console.log(`Error: ${error}`);
+  console.error(`Error: ${error}`);
```
Comment (yin1999) [inline files/zh-cn/mozilla/add-ons/webextensions/api/tabs/query/index.md:131]:
```suggestion
  console.error(`错误：${error}`);
```

File: files/zh-cn/mozilla/add-ons/webextensions/api/tabs/query/index.md [hunk 2]
```diff
@@ -1,151 +1,146 @@
...
+    - `windowId` {{optional_inline}}
+      - : `integer`。父窗口的 ID，或 {{WebExtAPIRef('windows.WINDOW_ID_CURRENT')}}（用于当前窗口）。
+    - `windowType` {{optional_inline}}
+      - : {{WebExtAPIRef('tabs.WindowType')}}。标签页所在窗口的类型。
 
-### Return value
+### 返回值
 
-A [`Promise`](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Promise) that will be fulfilled with an `array` of `{{WebExtAPIRef('tabs.Tab')}}` objects, containing information about each matching tab.
+一个 [`Promise`](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Promise) 将被兑现为一个包含 {{WebExtAPIRef('tabs.Tab')}} 对象（其中包含有关每个匹配的标签页的信息）的数组。
```
Comment (jasonren0403) [inline files/zh-cn/mozilla/add-ons/webextensions/api/tabs/query/index.md:63]:
```suggestion
一个将兑现为包含 {{WebExtAPIRef('tabs.Tab')}} 对象（其中包含有关每个匹配的标签页的信息）的数组的 [`Promise`](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Promise) 对象。
```

File: files/zh-cn/mozilla/add-ons/webextensions/api/tabs/query/index.md [hunk 3]
```diff
@@ -1,151 +1,146 @@
...
 
-let querying = browser.tabs.query({currentWindow: true, active: true});
-querying.then(logTabs, onError);
+browser.tabs
+  .query({ currentWindow: true, active: true })
+  .then(logTabs, onError);
 ```
 
-Get tabs for all HTTP and HTTPS URLs under or any of its subdomains: `"mozilla.org"`
+获取 `"mozilla.org"` 极其任何子域下所有 HTTP 和 HTTPS URL 的标签页：
```
Comment (jasonren0403) [inline files/zh-cn/mozilla/add-ons/webextensions/api/tabs/query/index.md:120]:
```suggestion
获取 `"mozilla.org"` 及其任何子域下所有 HTTP 和 HTTPS URL 的标签页：
```

**Review by jasonren0403 — COMMENTED** [review]
suggestions

**Review by jasonren0403 — APPROVED** [review]
👍

**Pull request merged**